pytorch_to_returnn/torch/nn/utils/__init__.py

```diff
@@ -1,2 +1,3 @@
 
+from .rnn import *
```

Comment:
This looks wrong. Why do you need this?
Maybe you just want this instead?
```suggestion
from . import rnn
```